bluepill/src/BPPacker.m

```diff
@@ -97,6 +97,41 @@ @implementation BPPacker
     return bundles;
 }
 
++ (NSMutableArray<BPXCTestFile *> *)splitXCTestBundles:(NSArray<BPXCTestFile *> *)xcTestFiles
```

Comment:
Where is the documentation for this function? If it's private, please add comments here, if public, please add it to the header with comments there